PORT-3737 action notification docs (#335)
port-labs/port-docs: docs/create-self-service-experiences/set-self-service-actions-rbac/examples.md

</TabItem>

</Tabs>

> ## Setting up a slack notification

Reviewer: ```suggestion
## Setting up a Slack notification
```

To enable Slack notifications, you need to create a Slack app and install it in your workspace by following steps 1-3 outlined in the [Slack API Documentation](https://api.slack.com/messaging/webhooks).

Once you have completed the installation process, you will obtain a webhook URL that looks like this:

```text
https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX
```

> Now, you can use your webhook URL to configure your approval notification to be sent to a Slack channel.

Reviewer: ```suggestion
Now, you can use your webhook URL to send the manual approval notification to the Slack channel tied to the webhook URL.
```

> To do this, modify the approvalNotification field in your action configuration as follows:

Reviewer: ```suggestion
To do this, modify the `approvalNotification` field in your action configuration as shown below:
```

```json showLineNumbers
{
  ...
  "requiredApproval": true,
  // highlight-start
  "approvalNotification": {
    "type": "webhook",

>     "format": "json / slack",

Reviewer: This is an example for slack, so the example should be specifically slack no?
